src/toolbar/components/toolbar-menu/toolbarListComponent.ts

```diff
@@ -52,6 +52,12 @@ export class ToolbarListController implements IToolbarListBindings {
       this.toolbarList.items &&
       this.toolbarList.items.filter((item: IToolbarItem) => !item.hidden).length > 0;
   }
+
+  private isToolbarEnabled(): boolean {
+    return this.toolbarList &&
```

Comment:
maybe add ` && this.toolbarList.enabled` to move the whole condition here?

Reply:
Yup, that sounds better, plus use `some` so we don't have to iterate over the whole array.
```TS
private isToolbarEnabled(): boolean {
  return this.toolbarList && 
    this.toolbarList.enabled &&
    this.toolbarList.items &&
    this.toolbarList.items.every((item: IToolbarItem) => item.enabled)
}
```